perun-base/pom.xml

```diff
@@ -136,6 +136,13 @@
 			<artifactId>javax.mail</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20140107</version>
+			<scope>compile</scope>
```

Comment:
``compile`` is default dependency type, you don't have to specify it.

Reply:
Removed